src/frontend/screens/AppPasscode/index.tsx

```diff
@@ -21,12 +22,12 @@ const m = defineMessages({
 
 export const AppPasscode: NavigationStackScreenComponent = () => {
   const { passIsSet } = React.useContext(SecurityContext);
-  const [{ experiments }] = React.useContext(SettingsContext);
   const { navigate } = useNavigation();
+  const { appPasscode } = devExperiments;
 
   React.useEffect(() => {
-    if (!experiments.appPasscode) navigate("Settings");
-  }, [experiments]);
+    if (!appPasscode) navigate("Settings");
+  }, [appPasscode]);
```

Comment:
this won't change in the React lifecycle so it shouldn't need to be specified in the deps array. i think something like the following would suffice?

```js
React.useEffect(() => {
  if (!devExperiments.appPasscode) navigate("Settings")
}, [])